File: src/main/java/com/cocktailsguru/app/common/dto/PagingDto.java
package com.cocktailsguru.app.common.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class PagingDto {
    private int pageNumber;
    private int pageSize;

    public int getPageNumber() {
        return pageNumber;
    }

    public int getPageSize() {
        return pageSize;
    }
}

File: src/main/java/com/cocktailsguru/app/common/dto/PagingDto.kt
package com.cocktailsguru.app.common.dto

import com.cocktailsguru.app.common.domain.PagingInfo

data class PagingDto(
        val pageNumber: Int,
        val pageSize: Int
) {
    constructor(pagingInfo: PagingInfo) : this(pagingInfo.pageNumber, pagingInfo.pageSize)
}
